drivers/flash/Kconfig

```diff
@@ -42,6 +42,33 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+choice FLASH_ERASE_BYTE_DEFAULT
+	prompt "Default flash erase value"
+	default FLASH_ERASE_BYTE_FF
+	help
+	  Select value that corresponds to value of erased flash cell.
+
+config FLASH_ERASE_BYTE_FF
+	bool "Erased flash value is 0xFF"
+
+config FLASH_ERASE_BYTE_00
+	bool "Erased flash value is 0x00"
+
+endchoice
```

Comment:
Not sure whether the selection is needed or the default value should be just hardcoded to 0xff. Please advice.

Reply:
I would resign from FLASH_ERASE_BYTE_VAL - just implement the API for all the driver in the codebase.